<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoodTracker | GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
Stylesheet copied from https://github.com/All4Gis/Load-QSS
*/

.QWidget {
   background-color: beige;
}

/* Nice Windows-XP-style password character. */
QLineEdit[echoMode=&quot;2&quot;] {
    lineedit-password-character: 9679;
}

/* We provide a min-width and min-height for push buttons
   so that they look elegant regardless of the width of the text. */
QPushButton {
    background-color: palegoldenrod;
    border-width: 2px;
    border-color: darkkhaki;
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}

QLabel, QAbstractButton {
    font: bold;
}

/* Mark mandatory fields with a brownish color. */
.mandatory {
    color: brown;
}

/* Bold text on status bar looks awful. */
QStatusBar QLabel {
   font: normal;
}

QStatusBar::item {
    border-width: 1;
    border-color: darkkhaki;
    border-style: solid;
    border-radius: 2;
}

QComboBox, QLineEdit, QSpinBox, QTextEdit, QListView {
    background-color: cornsilk;
    selection-color: #0a214c; 
    selection-background-color: #C19A6B;
}

QListView {
    show-decoration-selected: 1;
}

QListView::item:hover {
    background-color: wheat;
}

/* We reserve 1 pixel space in padding. When we get the focus,
   we kill the padding and enlarge the border. This makes the items
   glow. */
QLineEdit, QFrame {
    border-width: 2px;
    padding: 1px;
    border-style: solid;
    border-color: darkkhaki;
    border-radius: 5px;
}

/* As mentioned above, eliminate the padding and increase the border. */
QLineEdit:focus, QFrame:focus {
    border-width: 3px;
    padding: 0px;
}

/* A QLabel is a QFrame ... */
QLabel {
    border: none;
    padding: 0;
    background: none;
}

/* A QToolTip is a QLabel ... */
QToolTip {
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}

/* Nice to have the background color change when hovered. */
QRadioButton:hover, QCheckBox:hover {
    background-color: wheat;
}

/* Force the dialog's buttons to follow the Windows guidelines. */
QDialogButtonBox {
    button-layout: 0;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="6" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>X Quit X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="viewall_btn">
      <property name="text">
       <string>| View all Entries |</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="viewall_frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QTableWidget" name="viewall_table">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>DateTime</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mood</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Journal</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>&lt; Back &lt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="add_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="what_mood_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="moods_label">
            <property name="text">
             <string>Are you </string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>What is your mood right now?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mood_input"/>
          </item>
          <item>
           <widget class="QPushButton" name="next_btn">
            <property name="text">
             <string>&gt; Next &gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QFrame" name="why_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="why_feel_label">
            <property name="text">
             <string>Why do you feel </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="why_feel_input"/>
          </item>
          <item>
           <widget class="QPushButton" name="record_mood_btn">
            <property name="text">
             <string>+ Record your Mood +</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>viewall_btn</sender>
   <signal>clicked()</signal>
   <receiver>viewall_frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewall_btn</sender>
   <signal>clicked()</signal>
   <receiver>add_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewall_btn</sender>
   <signal>clicked()</signal>
   <receiver>pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>viewall_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>add_frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>viewall_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewall_btn</sender>
   <signal>clicked()</signal>
   <receiver>viewall_btn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pushButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_btn</sender>
   <signal>clicked()</signal>
   <receiver>why_frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>627</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_btn</sender>
   <signal>clicked()</signal>
   <receiver>what_mood_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>record_mood_btn</sender>
   <signal>clicked()</signal>
   <receiver>why_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>record_mood_btn</sender>
   <signal>clicked()</signal>
   <receiver>what_mood_frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewall_btn</sender>
   <signal>clicked()</signal>
   <receiver>why_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewall_btn</sender>
   <signal>clicked()</signal>
   <receiver>what_mood_frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
